sktime/transformations/series/simple_fourier.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+"""Simple Fourier transform for time series, interfacing numpy rfft."""
+
+__author__ = ["blazingbhavneek"]
+
+
+import numpy as np
+import pandas as pd
+from numpy.fft import rfft, rfftfreq
+
+from sktime.transformations.base import BaseTransformer
+
+
+class SimpleFourier(BaseTransformer):
+    r"""Simple Fourier transform for time series.
+
+    Fourier Transform is used to test data for periodicity.
+    Non-periodic/Noisy/Random data will have limited and
+    random amplitudes for frequencies, whereas periodic data
+    will have exceptional amplitudes for certain frequencies.
+
+    The implementation is based on the fast fourier transform from numpy.fft
+
+    Examples
+    --------
+    >>> from sktime.transformations.series.simple_fourier import SimpleFourier
+    >>> from sktime.datasets import load_airline
+    >>> X = load_airline()
+    >>> transformer = SimpleFourier()
+    >>> X_ft = transformer.fit_transform(X)
+    """
+
+    _tags = {
+        "scitype:transform-input": "pd.Series",
```

Comment:
This should be `Series`, not `pd.Series`, Same for the output.
`X_inner_mtype` should be `pd.Series`.

Reply:
Hey! I tried this, but it is giving me a `TypeError`, AND it is also returning an output

![image](https://user-images.githubusercontent.com/86467990/219357812-2aa27f46-eae3-417f-abcf-27f014813249.png)

Reply:
that won't work, if you look at the tag specification, in the extension template, it needs to be `Series`, `Panel`, or `Primitives`.
I think both should be `"Series"` (just the single string)

Reply:
Hey! So I expanded the `TypeError` to see whats wrong; Basically the issue was with index of the returned series, it shouldn't be float64 but RangedIndex, which is fine because the simple-interfacing anyways makes the index a ranged float (0-0.5 with size `X.size`)

![image](https://user-images.githubusercontent.com/86467990/219765844-1852c885-1fd8-48e7-950f-2f5981d0af78.png)

Reply:
yes, resetting the index would probably solve it.

Reply:
Should I make it limited to univariate only? Or add support for multivariate as well?

![image](https://user-images.githubusercontent.com/86467990/219848656-27c8275c-2e88-4ac2-888e-f3dddce98cdb.png)

Reply:
I would recommend to track it down where this is coming from - i.e., from the framework or from inside the function. I.e., debug using `check_estimator` with the `raise_exceptions` flag set, and have a look at the traceback.